vdb/recon/__init__.py

```diff
@@ -68,6 +68,21 @@ def reprargs(trace, fmt, args):
         r.append(rstr)
     return r
 
+def getArgs(trace, args):
+    '''
+    Assuming we are at the instruction after
+    a call, grab the argument at the specified
+    index (skipping the saved instruction pointer).
+    '''
+    cc = trace.getEmulator().getCallingConvention("stdcall")
+    if "64" in trace.getMeta("Architecture"):
```

Comment:
Love the overall idea, however for future proofing, we cannot assume that any arch with the chars "64" in the name ( such as commodore64 ) are amd64.  This probably needs to be come something more like a dict lookup of
(arch,plat):cc